src/art/betiniakarandut/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title></title>
```

Comment:
Please add a title in the metatag

Reply:
I have added a title and pushed to the remote repo. Please confirm it. Thank you